### PR TITLE
AO landing styles

### PR DESCRIPTION
**Pull request by noahmanger**
Adds a couple styles to support the new AO landing page in https://github.com/18F/openFEC-web-app/pull/1731:
- Styles for the titles of pending AOs
- Styles for the little document link
- Adjusts the sizing of the search container

### CODE REVIEW

**Comment by jenniferthibault** [comment]
👍 May want technical review from @xtine

File: scss/components/_options.scss [hunk 1]
```diff
@@ -41,6 +41,12 @@
   padding-top: u(2rem);
 }
 
+.option__content {
+  .label {
+    margin-bottom: 2rem;
```
Comment (xtine) [inline scss/components/_options.scss]: `u(2rem)`?

File: scss/components/_contact-items.scss [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
   @include clearfix();
+  display: table;
   margin-bottom: u(4rem);
+
+  &:not(.contact-item--half) {
+    &::before {
+      content: '';
+      display: block;
+      background-size: 90%;
+      margin-right: 2rem;
```
Comment (xtine) [inline scss/components/_contact-items.scss]: Just noticed a few more units not in the `u` function in these styles.

File: scss/components/_posts.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 .post-feed {
   border-top: 1px solid $gray;
+  border-bottom: 1px solid $gray;
   padding: u(1rem 0 1rem 2rem);
```
Comment (xtine) [inline scss/components/_posts.scss]: I made a note in https://github.com/18F/openFEC-web-app/pull/1731 about the `padding-left` indent. Just want to make sure if it's intentional or needs a fix.

Comment (noahmanger) [inline scss/components/_posts.scss]: Yep, that's the style we're going with as it's used on the press landing page too.

File: scss/components/_contact-items.scss (hunk 2 above)
Comment (noahmanger) [inline scss/components/_contact-items.scss]: Fixed!

**Comment by noahmanger** [comment]
Fixed merge conflicts. We were no longer using the `.icon-heading--calendar-circle` class elsewhere, so I removed it and un-nested the one you made for the home page.

**Pull request merged**